stripecardscan/src/main/java/com/stripe/android/stripecardscan/cardimageverification/CardImageVerificationActivity.kt

```diff
@@ -138,6 +139,11 @@ internal open class CardImageVerificationActivity :
      */
     private var requiredCardLastFour: String? = null
 
+    /**
+     * The image format to send to verify_frames
+     */
+    private var imageConfigs: AcceptedImageConfigs = AcceptedImageConfigs()
```

Comment:
this is absolutely the right place to put this. It's a bummer we have state in our activities. I'd like to eventually move this stuff to a view model or more generalized scan state.